docs/controller-skills/sequencing.md

```diff
@@ -8,12 +8,88 @@ title: Sequencing
 
 It's all well and good maintaining separation standards in your airspace, but if you handoff 2 aircraft to ML_APP overhead LIZZI at the same speed, 1000ft apart, you're not going to make any friends. By default, aircraft must be sequenced **2 minutes** apart at the **Feeder Fix** (generally the first waypoint of the STAR).  
 
-It's important to remember that aircraft aren't just arriving from your sector, they can be coming in from all directions at similar times. **MAESTRO** is a tool used to help aid with sequencing (what time do YOU need to get the aircraft to the feeder fix to avoid TCU congestion). You can find more info [here](../../controller-skills/maestro)
+It's important to remember that aircraft aren't just arriving from your sector, they can be coming in from all directions at similar times. **MAESTRO** is a tool used to help aid with sequencing (what time do YOU need to get the aircraft to the feeder fix to avoid TCU congestion). You can find more info [here](../maestro).
 
 In real life, controllers have the luxury of putting a fair bit of the onus of forming the sequence on the pilots, by issuing instructions like *"Adjust speed to cross RIVET at time 52 at 250kts in to published speeds"*. On VATSIM, every pilot may be using a different clock, potentially even different weather, and the pilot may simply not be proficient enough to meet a FF time. As such, all sequencing is best left in the hands of you, the controller.
 
 There are multiple tools a controller can use to create and maintain a sequence.
 
+## Arrivals List
+The Arrivals List window is a sequencing management tool that can be used side-by-side with, or in-lieu of, [Maestro](../maestro).
+
+It is a **global** window that can be viewed and manipulated by Tower, TCU, and Enroute Controllers.
+
+The responsibility for managing the window falls on the:
+
+- **Flow** Controller
+    - This is delegated to the **Approach** Controller when Flow is Offline
+- **ADC** Controller at [Procedural Towers](../../aerodromes/procedural/)
+
```

Comment:
Could we throw a note in here that the ADC (or TMA if top down) controller should move aircraft back above the bar as they land?

Reply:
Could do, but is it required? I believe the strip disappears when the aircraft disconnects from the network, is that sufficient? I believe IRL it disappears when the plane lands so i guess there's a bit more of a gap there if we wanna do that